_includes/notebook_colab_link.html

```diff
@@ -0,0 +1,5 @@
+<div class="px-2">
+    <a style="border:0;" href="https://colab.research.google.com/github/{{site.github_username}}/{{site.github_repo | default: site.baseurl | remove: "/" }}/blob/{{page.branch | default: "master"}}/{{page.nb_path}}">
```

Comment:
Do we still ship a sass / scss file with some styles by default? If we do, maybe moving that border style in it would be a good idea to not have an inline css?

Reply:
Good idea